R/load-blueprints.R

```diff
@@ -71,8 +69,15 @@ fetch_blueprint_files <- function(directory) {
 }
 
 import_blueprint_file <- function(bp_file, env = parent.frame()) {
+  # Add reference to blueprint script location to ease custom
+  # metadata location writing. For example, rather than specifying the full
+  # `metadata_file_path`, one could just set the `blueprintr.use_local_metadata_path`
+  # option to `TRUE` and then just use `metadata_file_name` to set the
+  # file base name.
+  env$cur_blueprint_script <- bp_file
```

Comment:
Remove or change this comment for clarity that it operates on both `metadata_directory` _and_ `metadata_file_name`.